project/interface.go

```diff
@@ -37,6 +37,7 @@ type APIProject interface {
 	CreateService(name string) (Service, error)
 	AddConfig(name string, config *config.ServiceConfig) error
 	Load(bytes []byte) error
+	ListStoppedContainers(ctx context.Context, services ...string) ([]string, error)
```

Comment:
Adding more container logic to `project` makes it a bit trickier to use without the `docker` package. Certain container operations are difficult to implement with orchestration systems.

For example, I've never really been able to do a proper implementation of `IsRunning`. As a result, `Delete` didn't really work since it'll always say that there are no containers stopped.

I'm definitely not opposed to adding functions to the `Container` interface, but it would be helpful to gracefully handle the case where they're not implemented :smile: